scripts/Install-EndpointAgentTcpNetworkTests.ps1

```diff
@@ -195,10 +195,10 @@ process {
 
     # Default feature states for optional features
     $proposedFeatures = [ordered]@{
-        "TcpNetworkTestsSupport" = $false
-        "IeExtension"            = $false
-        "ChromeExtension"        = $false
-        "EdgeExtension"          = $false
+        "ExtendedNetworkTestsSupport"   = $true
```

Comment:
I think this needs a bit more work. First, we should rename the script name to replace `*Tcp*` with `*Extended*`. Secondly, does this work if the customer has installed an older agent previously with `TcpNetworkTestsSupport = $false`? If not, this script should handle this case as well: i.e, it should enable the extended tests support feature and install npcap regardless of any previous settings. Can you please test this @chrislawr-te ?

Reply:
I can confirm, Npcap was installed regardless of the previous installers settings

<img width="1450" alt="Screenshot 2023-01-24 at 16 23 18" src="https://user-images.githubusercontent.com/98470851/214349936-eed93e47-2a88-4e23-b844-030cc496c558.png">

Reply:
I upgraded from 1.143.100 to 1.146.100 (my modified dev version)

Reply:
Added [PR](https://github.com/thousandeyes/eyebrow/pull/1314) to update tool in Eyebrow, when merged I can then copy of the changes to this branch

Reply:
It does this without any change? How is this possible since in your original eyebrow PR's upgrade scenario (where npcap is *not* installed), upgrading with the default settings does not affect the (lack of) npcap installation.

Can you retest the original upgrade scenario? What's the difference between e.g., upgrading interactively (or in `/quiet` mode) from `1.143.100` to `1.146.100` vs running this script to make the same upgrade?

Reply:
@ggeorgalis @jerriley-te I ran the two scenarios again interactive and quiet - The interactive respects the previous installer decision (to not install Npcap)  running the script (/quiet) Npcap is forcibly installed

Reply:
I haven't examined it, but I presume that the code forces the feature on in this script will override the installer configuration that sets in to off in the upgrade with no-tcp-tests selected scenario.